server/e2e_test_command.go

```diff
@@ -15,7 +15,6 @@ import (
 const (
 	e2eTestMsgCommenterPermission = "You don't have permissions to trigger this command.\n It's only available for organization members."
 	e2eTestMsgCIFailing           = "The command /e2e-test requires all PR checks to pass."
```

Comment:
Is there no potential cyclic issue when this message is commented my mattermod? I wonder if this message could be made more generic like "The slash command requires all PR checks to pass.". Though it might not be obvious which slash command it's referring to.

Reply:
Oh my, yes, this is what is causing it :face_palm: ! Thanks! 
Still doesn't explain why the tests didn't catch it. I will fix this.